### PR TITLE
Update motd to reflect new organization name.

### PR DESCRIPTION
**Pull request by felddy**
Used `toilet -F gay -f kban CISA > motd` and added the C+I text to 
bottom by hand.

### CODE REVIEW

**Comment by felddy** [comment]
motd now renders like this on login:
image

File: files/motd [hunk 1]
```diff
@@ -1,6 +1,6 @@
-'|.   '|'   ..|'''.|     |     |''||''|  .|'''.|  
- |'|   |  .|'     '     |||       ||     ||..  '  
- | '|. |  ||           |  ||      ||      ''|||.  
- |   |||  '|.      .  .''''|.     ||    .     '|| 
-.|.   '|   ''|....'  .|.  .||.   .||.   |'....|'  
-
+  [0;1;31;91m..[0;1;33;93m|'[0;1;32;92m''[0;1;36;96m.|[0m [0;1;34;94m'[0;1;35;95m||[0;1;31;91m'[0m  [0;1;33;93m.[0;1;32;92m|'[0;1;36;96m''[0;1;34;94m.|[0m      [0;1;32;92m|[0m
```
Comment (jsf9k) [inline files/motd]: Since you added that sweet plain text, we could just check for that right?  That should be good enough, no?

**Review by jsf9k — APPROVED** [review]
The strongest of works.

**Comment by h-m-f-t** [comment]
> motd now renders like this on login:
> image

This is really cool.

**Pull request merged**